multiversx_sdk_cli/tests/test_cli_all.sh

```diff
@@ -3,15 +3,14 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 USE_PROXY=$1
 
+source ./shared.sh
+source ./test_cli_validators.sh && testAll
```

Comment:
not sure why this is here and not in the `testAll` function

Reply:
removed.